src/Request/Request.ts

```diff
@@ -0,0 +1,11 @@
+export interface RequestParams {
+	id: string;
+}
+
+export interface ResponseBody {}
+
+export interface RequestBody {}
```

Comment:
This module's (and all its interfaces) naming isn't too descriptive IMO, since there are more than a single type of request (e.g. login).

Reply:
these interfaces are replaced by @Teg64's validators in frontend port

Reply:
Merged and replaced